talent/job-descriptions/offers/data-retrieval.md

```diff
@@ -9,15 +9,15 @@ Software Engineer in the Data Retrieval team
 
 ### [Engineering at source{d}](../engineering-section.md)
 
-### Role 
+### Role
 
-The Data Retrieval team is developing source{d}'s high-level Engine for running scalable data retrieval pipelines that process and manipulate any number of code repositories for source code analysis. Written mostly in Scala, it aims to be robust, friendly and flexible: it is built on top of Apache Spark, accessible both via Scala and Python Spark APIs, and capable of running on large-scale distributed clusters over petabytes of data.
+The Data Retrieval team is developing source{d}'s high-level Engine for running scalable data retrieval pipelines that process and manipulate any number of code repositories for source code analysis. Written mostly in Go, it aims to be robust, friendly and flexible: it is built on top of Apache Spark, accessible both via Scala and Python Spark APIs, and capable of running on large-scale distributed clusters over petabytes of data.
```

Comment:
I think we should rethink this paragraph.

Reply:
@ajnavarro I made some changes based on your comments, if you still think it needs to be improved we can do it together.